src/Stratis.Bitcoin.Tests.Common/TestBase.cs

```diff
@@ -197,9 +197,9 @@ public PosBlock CreatePosBlock()
         /// </summary>
         /// <param name="blockCount">The amount of blocks to chain.</param>
         /// <returns>Tip of a created chain of headers.</returns>
-        public ChainedHeader BuildProvenHeaderChain(int blockCount)
+        public ChainedHeader BuildProvenHeaderChain(int blockCount, ChainedHeader currentHeader = null)
```

Comment:
Please add comment for the `currentHeader`. 

Also I don't think it's a good name for a variable, it gives no info on what is this header